.jshintrc

```diff
@@ -14,6 +14,7 @@
   "globals": {
     "angular": true,
     "_": true,
+    "$": true,
```

Comment:
jQuery was already in the project. Probably for the bootstrap plugins in to work. Which I don't think you ever used initially. But i set that so i could use jQuery for a date picker plugin which is in the booking controller.